continuous_load_testing/client-go-manual-cloudpath.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: client-go-manual-cloudpath
+  name: client-go-manual-cloudpath
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: client-go-manual-cloudpath
+  template:
+    metadata:
+      labels:
+        app: client-go-manual-cloudpath
+    spec:
+      containers:
+      - image: "us-docker.pkg.dev/directpathgrpctesting-client/directpathgrpctesting-client/directpathgrpctesting-client-go-manual"
+        imagePullPolicy: Always
+        name: client-go-manual-cloudpath
+        env:
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: NAMESPACE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: CONTAINER_NAME
+          value: client-go-manual-cloudpath
+        - name: OTEL_RESOURCE_ATTRIBUTES
+          value: k8s.pod.name=$(POD_NAME),k8s.namespace.name=$(NAMESPACE_NAME),k8s.container.name=$(CONTAINER_NAME)
+        - name: GOOGLE_CLOUD_DISABLE_DIRECT_PATH
```

Comment:
Can we set the environment variable in the docker's cmd script? We don't want to maintain and open-ended list of yaml files to run different kinds of manual workload. It's supposed to modify the manual yaml locally (such as changing cmd arguments, env var) to start the manual test workload.
In the future, we should add the user name automatically when creating the manual workload (by using ${user}, etc), thus people won't interfere each other.